modules/schem/src/main/java/net/hollowcube/util/schem/Schematic.java

```diff
@@ -50,6 +54,36 @@ public Point offset(Rotation rotation) {
         return rotatePos(offset, rotation);
     }
 
+    public @NotNull CompletableFuture<Void> applyToInstance(@NotNull Instance instance, @NotNull Point startPoint, boolean loadChunks, @NotNull Rotation rotation, @Nullable Function<Block, Block> blockModifier) {
+        RelativeBlockBatch batch = build(rotation, blockModifier);
+        if (loadChunks) {
+            Point min = startPoint.add(offset());
```

Comment:
you could also specify the rotation here, by adding a parameter to the function, as `offset()` can take a rotation parameter.